@jianghaolu There is some error with the node.js CI job itself I think.

```
npm ERR! code EINTEGRITY
npm ERR! sha512-CRxb97Ib3b6GWpF2bE37Vvqtd5KfsZ912OBWe0k92Zvk8LxY4WcQu4C6F4Numm78m8FdIWTAH4u9uT91riVdxQ== integrity checksum failed when using sha512: wanted sha512-CRxb97Ib3b6GWpF2bE37Vvqtd5KfsZ912OBWe0k92Zvk8LxY4WcQu4C6F4Numm78m8FdIWTAH4u9uT91riVdxQ== but got sha512-ouR0DO5JVkiOlGb9FenjY8svbvOTL0bkDnyJGcnZy9hrQxGjVzWO2Y3VOvQh8jCkAvBjaTKZMjWAcp6Fq1K2mA==. (2228224 bytes)
```

--- a/specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/KeyVault.json
+++ b/specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/KeyVault.json
@@ -65,11 +65,11 @@
             "description": "Contains the localized display information for this particular operation / action.",
             "properties": {
                 "provider": {
-                    "description": "The localized friendly form of the resource provider name � it is expected to also include the publisher/company responsible. It should use Title Casing and begin with \"Microsoft\" for 1st party services. e.g. \"Microsoft Monitoring Insights\" or \"Microsoft Compute.\"",
+                    "description": "The localized friendly form of the resource provider name; it is expected to also include the publisher/company responsible. It should use Title Casing and begin with \"Microsoft\" for 1st party services. e.g. \"Microsoft Monitoring Insights\" or \"Microsoft Compute.\"",
                     "type": "string"
                 },
                 "resource": {
-                    "description": "The localized friendly form of the resource type related to this action/operation � it should match the public documentation for the resource provider.",
+                    "description": "The localized friendly form of the resource type related to this action/operation; it should match the public documentation for the resource provider.",
                     "type": "string"
                 },
                 "operation": {

--- a/specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/Quotas.json
+++ b/specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/Quotas.json
@@ -61,7 +61,6 @@
                 "properties": {
                     "x-ms-client-flatten": true,
                     "description": "Properties for keyvault quotas.",
-                    "value": null,
                     "$ref": "#/definitions/QuotaProperties"
                 }
             },

